src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -66,19 +66,25 @@ public static void Initialize()
                 return;
             }
 
+            Log.Debug("Initialization started.");
+
             try
             {
-                // ensure global instance is created if it's not already
-                _ = Tracer.Instance;
+                Log.Debug("Sending CallTarget integrations definitions to native profiler.");
```

Comment:
Let's try to not use "profiler" to avoid confusion with the Continuous Profiler product. We can say the tracer has a managed library and a native library. Or maybe "component" instead of "library"?
```suggestion
                Log.Debug("Sending CallTarget integrations definitions to native library.");
```

Reply:
done in 73f5a81